workflow/hosts/jet.yaml

```diff
@@ -25,4 +25,4 @@ SUPPORTED_RESOLUTIONS: ['C384', 'C192', 'C96', 'C48']
 COMINecmwf: /mnt/lfs5/HFIP/hfv3gfs/glopara/data/external_gempak/ecmwf
 COMINnam: /mnt/lfs5/HFIP/hfv3gfs/glopara/data/external_gempak/nam
 COMINukmet: /mnt/lfs5/HFIP/hfv3gfs/glopara/data/external_gempak/ukmet
-AERO_INPUTS_DIR: /lfs5/HFIP/hfv3gfs/glopara/data/gocart_emissions
+AERO_INPUTS_DIR: /lfs5/HFIP/hfv3gfs/glopara/data/GEFS_ExtData/20250310
```

Comment:
Note and an FYI, this does not exist since we are dropping support for Jet. Fine to update this here though.